model/lora-net-device.cc

```diff
@@ -195,7 +195,7 @@ LoraNetDevice::IsLinkUp (void) const
 {
   NS_LOG_FUNCTION (this);
 
-  return m_phy != 0;
+  return bool(m_phy);
```

Comment:
I left the bool cast here because it was still giving me a warning

Reply:
Here I would suggest using return `(m_phy != nullptr)`, for the sole reason that the ns-3 coding style guidelines are agains unnecessary casts, and whenever those are needed to use C++ style casts instead of C ones.